techniques/flash/FLASH24.html

```diff
@@ -18,7 +18,7 @@ <h3>Changing timeout with a dropdown list</h3>
     						by the user through a dropdown list. In this example there is a combobox
     						with the instance name sessionLimitDuration. </p>
          
-         <pre xml:space="preserve">import mx.controls.Alert;
+         <pre>import mx.controls.Alert;
```

Comment:
all the naked `<pre> ... </pre>` blocks in this technique could do with inner `<code> ... </code>`